src/Optimization/hiopKKTLinSysSparseCondensed.hpp

```diff
@@ -208,26 +216,39 @@ class hiopKKTLinSysCondensedSparse : public hiopKKTLinSysCompressedSparseXDYcYd
                                 const double& delta_cc,
                                 const double& delta_cd);
 
-  virtual bool solveCompressed(hiopVector& rx, hiopVector& rd, hiopVector& ryc, hiopVector& ryd,
-                               hiopVector& dx, hiopVector& dd, hiopVector& dyc, hiopVector& dyd);
+  virtual bool solveCompressed(hiopVector& rx,
+                               hiopVector& rd,
+                               hiopVector& ryc,
+                               hiopVector& ryd,
+                               hiopVector& dx,
+                               hiopVector& dd,
+                               hiopVector& dyc,
+                               hiopVector& dyd);
 protected:
-
+  virtual bool solve_compressed_direct(hiopVector& rx,
```

Comment:
maybe it's a good idea to add a brief description to this function and the above one (even though I know the difference :-))